chain/src/chain.rs

```diff
@@ -504,7 +504,7 @@ impl Chain {
 	/// spent. This querying is done in a way that is consistent with the
 	/// current chain state, specifically the current winning (valid, most
 	/// work) fork.
-	pub fn is_unspent(&self, output_ref: &OutputIdentifier) -> Result<CommitPos, Error> {
+	pub fn is_unspent(&self, output_ref: &OutputIdentifier) -> Result<Option<CommitPos>, Error> {
```

Comment:
Does this now return `None` if an output is spent? 
Might be worth refactoring this and naming it `get_unspent()` rather than `is_unspent()`?

Reply:
Yes, agreed. Will update